examples/thing/cert-registration/main.go

```diff
@@ -112,21 +111,9 @@ func certRegThing() (err error) {
 	builder := thing.New().
 		ConnectTo(u).
 		InRealm(*realm).
-		AuthenticateWith(*authTree).
-		HandleCallbacksWith(
-			callback.AuthenticateHandler{
-				Realm:   *realm,
-				ThingID: *thingName,
-				KeyID:   *keyID,
-				Key:     key},
-			callback.RegisterHandler{
-				Realm:        *realm,
-				ThingID:      *thingName,
-				ThingType:    callback.TypeDevice,
-				KeyID:        *keyID,
-				Key:          key,
-				Certificates: certs,
-			})
+		WithTree(*authTree).
+		AuthenticateThing(*thingName, *keyID, key, nil).
```

Comment:
Can we change this example to use the suggested `JWKThumbprint` function for the keyID?